clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -504,7 +495,7 @@ class GCEUtil {
   }
 
   static Map<String, String> buildMapFromMetadata(Metadata metadata) {
-    def map = metadata?.items?.collectEntries { Metadata.Items metadataItems ->
+    def map = metadata?.items?.collectEntries { def metadataItems ->
```

Comment:
The `metadata.items` returned from the google cluster provider is of type `Map`. This allows it to work either way.